core/src/main/java/org/keycloak/representations/idm/CredentialMetadataRepresentation.java

```diff
@@ -0,0 +1,43 @@
+package org.keycloak.representations.idm;
+
+public class CredentialMetadataRepresentation {
```

Comment:
Minor: IMO it will be slightly better to move this class into package `org.keycloak.representations.account` as it's main usage is for account REST API and account console right now

Reply:
OK.